README.md

```diff
@@ -26,7 +26,7 @@ Attempt to follow both the [Julia Contribution Guidelines](https://github.com/Ju
 When convention guidelines conflict this guide takes precedence (known conflicts will be noted in this guide).
 
 - Use 4 spaces per indentation level, no tabs.
-- Try to adhere to a 92 character line length limit.
+- Try to adhere to a 110 character line length limit.
```

Comment:
Can you add a blurb or comment about why it's 110 characters?  It would make it easier to update in the future

(or just a blurb in the MR)